pkg/cmd/server/start/start_master.go

```diff
@@ -460,6 +460,7 @@ func (m *Master) Start() error {
 			*m.config,
 			kubeControllerManagerConfig.KubeControllerManagerConfiguration.LeaderElection,
 			kubeExternal,
+			clientGoKubeExternal.Core().Events(""),
```

Comment:
So which is correct here: `Core` or `CoreV1`?  I feel like last time I used `CoreV1` I got `unregistered type event v1` errors...